src/Test.UnitTests.Sarif/Baseline2/V2ResultMatcherTests.cs

```diff
@@ -237,6 +237,7 @@ public void V2ResultMatcher_WhenNewIssueIsSuppressed_SupressesTheResultInTheOutp
             Result newSuppressedResult = matchedRun.Results.Single(r => r.Message.Text == "New suppressed result.");
 
             newSuppressedResult.Suppressions.Count.Should().Be(1);
+            AssertBaselinedRunInvariants(matchedRun);
```

Comment:
>AssertBaselinedRunInvariants [](start = 12, length = 28)

Terminology: are you using "baselined run" and "matched run" to mean the same thing? If so, then consider (on the "Ubiquitous Vocabulary" principle) using one or the other.